python-package/mlbox/encoding/categorical_encoder.py

```diff
@@ -455,6 +455,12 @@ def transform(self, df):
 
                 elif (self.strategy == 'entity_embedding'):
 
+                    def get_embeddings(x, col):
```

Comment:
@NCherel LGTY?

Not sure whether to use `self` or not.
`def get_dummies(self, x, col)` The problem with that is that it would shadow the outer `self`. Wouldn't it?